src/utilities/kumaDpServerUrl.ts

```diff
@@ -1,5 +1,5 @@
 export function kumaDpServerUrl(): string {
-  const dpServerUrl = String(import.meta.env.VITE_KUMA_DP_SERVER_URL)
+  const dpServerUrl = String(import.meta.env.VITE_KUMA_DP_SERVER_URL) + '/'
```

Comment:
Do you know why this is wrapped in a `String`, tiny nit, but would this be fine as a template literal?

Reply:
Historically, environment variables where types as something like `string | undefined` to honor the fact that accessing `process.env.STUFF` resolves to `undefined` if the variable isn’t actually defined.

Now, we have explicit types for the variables that are always defined and this can be dropped.

Also, I prefer not to use template literals if they don’t add any value. Here, the concatenation is neither more clear or shorter with template literals. Regular string concatenation is just fine.